Cargo.toml

```diff
@@ -28,5 +28,6 @@ tokio = { version = "1.37", features = ["full"] }
 tracing = "0.1"
 tracing-futures = { version = "0.2", features = ["futures-03"] }
 
-# [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "main" }
+[patch."https://github.com/stackabletech/operator-rs.git"]
+#stackable-operator = { path = "../operator-rs/crates/stackable-operator" }
+stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "main" }
```

Comment:
[stackable-operator-0.69.0](https://github.com/stackabletech/operator-rs/releases/tag/stackable-operator-0.69.0) is out :)

Reply:
thanks https://github.com/stackabletech/hdfs-operator/pull/534/commits/924f49b3baf0ac79be67604bb808f83b7b789863